src/components/displayDocuments/documentCard.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import { Document, Tag } from "./documents";
+import { DisplayDocuments } from "../displayDocuments/displayDocuments";
+
+const sampleDocuments: Document[] = [
+  {
+    title: "test1",
+    url: "test1",
+    imageName: "",
+    tags: [Tag.TRADETRUST]
+  }
+];
+
+describe("displayDocuments", () => {
+  it("should show Document, before QrCodeButton button is pressed", () => {
+    expect.assertions(1);
+    const { queryByTestId } = render(<DisplayDocuments documents={sampleDocuments} />);
+    expect(queryByTestId("document-container")).not.toBeNull();
+  });
+
+  it("should not show document, after QrCodeButton button is pressed", () => {
```

Comment:
I dont see the difference with the following test `should hide QrCodeButton and show QrCode, when QrCodeButton button is pressed`

I think you can merge test expectations it's fine: one scenario, multiple assertions is always fine